Assets/MixedRealityToolkit.Examples/Experimental/HandMenuLayout/Textures/gradient_1.png.meta

```diff
@@ -0,0 +1,88 @@
+fileFormatVersion: 2
+guid: 80c1e1d0180de774980573f1aca2de4f
```

Comment:
What are the gradients used for?

Reply:
Good catch, I was previously using them to drive tile icon colors, but opted for just flat color instead. Removed.